v2/event/content_type.go

```diff
@@ -14,6 +14,20 @@ const (
 	ApplicationCloudEventsBatchJSON = "application/cloudevents-batch+json"
 )
 
+type ContentType string
+
+// IsJSON returns true if the content type is a JSON type.
+func (c ContentType) IsJSON() bool {
```

Comment:
Instead of introducing an exported type, how about:

```suggestion
func isJSON(contentType string) bool { ... }
```

Reply:
The unit test cannot access it then :(

Reply:
I can ditch the unit test if preferred

Reply:
@embano1 updated as suggested, thanks 🙇